site/en/docs/extensions/reference/printing/index.md

```diff
@@ -97,4 +97,60 @@ chrome.printing.submitJob(submitJobRequest, (response) => {
 });
 ```
 
+### Roll printing {: #roll-printing }
+
+This example shows how to build a printer ticket for continuous (or roll) printing, which is often used with receipt printing. The `submitJobRequest` object for roll printing is the same as that shown for the [`submitJob()`](#submitjob) example.
+
+If you need to change the default value for paper cutting, use the `vendor_ticket_item` key. (The default varies from printer to printer.) When included, this key needs to be an array with one member: an object whose `id` is `'finishings'`. The value can either be `'trim'` for printers that cut the roll at the end of printing or `'none'` for printers that require the print job to be torn off.
+
+```json/3
+var ticket = {
+  version: '1.0',
+  print: {
+    vendor_ticket_item = [{id: 'finishings', value: 'trim'}],
+    color: {type: 'STANDARD_MONOCHROME'},
+    duplex: {type: 'NO_DUPLEX'},
+    page_orientation: {type: 'PORTRAIT'},
+    copies: {copies: 1},
+    dpi: {horizontal_dpi: 300, vertical_dpi: 300},
+    media_size: {
+      width_microns: 72320,
+      height_microns: 100000
+    },
+    collate: {collate: false}
+  }
+};
+```
 
+Some printers do not support the `"finishings"` option. To determine if your printer does, call [`getPrinterInfo()`](#method-getPrinterInfo) and look for a `"display_name"` of `"finishings/11"`.
+
+```json/2
+"vendor_capability": [
+  {
+    "display_name": "finishings/11",
+    "id": "finishings/11",
```

Comment:
Maybe the id can be different from the display name?

Reply:
These are vendor defined. So we can't change them.